recipes-devtools/arm-compute-library/arm-compute-library_24.04.bb

```diff
@@ -1,11 +1,11 @@
 SUMMARY = "The ARM Computer Vision and Machine Learning library"
 DESCRIPTION = "The ARM Computer Vision and Machine Learning library is a set of functions optimised for both ARM CPUs and GPUs."
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=a436777748fb1193895c051cea504ab6"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=44ab5c3155cc6931b2798b49c354b7bd"
```

Comment:
What changed here? Please include a License-Update tag in commit message.

Reply:
done